---
layout: post
title: The cancer cell cycle
date: 2008-10-22 13:17:32.000000000 -05:00
categories:
- genomics
tags:
- genomics
- health
- science
status: draft
type: post
published: false
meta:
  _edit_last: '2'
author:
  login: dd
  email: dooling@gmail.com
  display_name: dd
  first_name: David
  last_name: Dooling
excerpt: !ruby/object:Hpricot::Doc
  options: {}
---
<p>In cancer, the ability of a cell to regulate its division and therefore growth of the tissue is broken and the tissue, or tumor in this case, grows uncontrollably.  The loss of the cells ability to regulate its division is generally due to one or more protein that somehow plays a role in cell division being altered in such a way that it no longer functions properly.  Since proteins are generated by DNA transcription, the alteration of a protein is due to some mutation in the part of the genome that encodes that protein, i.e., the gene that transcribes that protein.  That mutation can be the change of a single nucleotide, e.g., an A becomes a G, or an insertion or deletion of some DNA into the gene sequence.  The result of the mutation could have no effect on the work of that protein or it could prevent the protein from doing its job.  If the job of the protein was to control when the cell began its division, uncontrolled division could take place in that cell and all its progeny since they too would contain the deleterious mutation.  Now, cell replication processes are quite complex and involve a complicated network of proteins and their interactions.  Therefore, it is not possible to predict /a priori/ what gene or genes were mutated that lead to tumorigenesis.  To figure out which genes might be involved, researchers often measure the amount of <a href="http://en.wikipedia.org/wiki/MRNA">mRNA</a> corresponding to each gene (and therefore protein) in normal and tumor cells and look for differences (mRNA is one of the intermediate steps in the transcription of DNA into a protein).  Genes which showed large differences in "expression" (transcription) between normal and tumor tissues were targeted as potential causative agents in the tumors genesis and growth.  Currently, expression arrays are used to measure relative levels of gene expression in tissues.  These arrays contain tags of DNA complimentary to specific genes, or more specifically specific <a href="http://en.wikipedia.org/wiki/Exon">exons</a> in genes.  <a href="http://en.wikipedia.org/wiki/CDNA">cDNA</a> is made from the mRNA (think of this as reverse transcription) and the cDNA is hybridized onto the array.  The degree of hybridization of each probe is measured and you obtain relative expression of each exon (gene).  What you do not get is any indication as to what mutation might have caused the expression to increase or decrease in the tumor sample as compared to the normal.  To get that information, you would have to selectively sequence that portion of the genome (usually a few hundred to a few thousand bases per probe).  The sequencing is expensive because of lower resolution probes, you may find lots of exons with discrepant expression rates, and this type of sequencing is still primarily done with more expensive, capillary-based sequencing methods.  What UHTS provides is a way to short cut this approach.  You can directly sequence the cDNA, thereby obtaining both the sequence and the relative expression levels.  The latter is obtained by aligning each sequence, or "read", generated against all genes.  The genes that have more reads aligned to them are expressed more than those that have fewer reads align to them.</p>
