---
layout: post
title: How much?
date: 2008-06-20 13:11:06.000000000 -05:00
categories:
- genomics
tags:
- '454'
- genomics
- Illumina
- informatics
- science
- SOLiD
- wustl
status: publish
type: post
published: true
meta:
  _edit_last: '2'
author:
  login: dd
  email: dooling@gmail.com
  display_name: dd
  first_name: David
  last_name: Dooling
excerpt: !ruby/object:Hpricot::Doc
  options: {}
---
<p><a href="http://genome.wustl.edu/">The Genome Center</a> recently published a paper entitled <a href="http://www.biomedcentral.com/1471-2105/9/239">Aspects of coverage in medical DNA sequencing</a> that develops a model for diploid sequence coverage using data from massively parallel sequencing platforms (454, Solexa, SOLiD). It uses a known yardstick, 8&times; BAC or WGS coverage with capillary sequencing, to establish the equivalent coverage for the new sequencing platforms. It turns out you need about 20&times; to 30&times; redundancy using these new platforms to obtain the equivalent amount of information as 10&times; coverage with capillary sequencing. The paper is published in an open access journal, <a href="http://www.biomedcentral.com/bmcbioinformatics">BMC Bioinformatics</a>, so enjoy.</p>
