---
layout: post
title: C. elegans revisited
date: 2008-02-14 10:30:00.000000000 -06:00
categories:
- genomics
tags:
- genomics
- Illumina
- informatics
- science
- wustl
status: publish
type: post
published: true
meta:
  blogger_blog: politigenomics.blogspot.com
  blogger_author: dd
  blogger_permalink: "/2008/02/c-elegans-revisited.html"
  _edit_last: '1'
author:
  login: admin
  email: ddgenome@gmail.com
  display_name: admin
  first_name: ''
  last_name: ''
excerpt: !ruby/object:Hpricot::Doc
  options: {}
---
<p><a href="http://genome.wustl.edu/">The Genome Center</a> recently published the <span style="font-weight:bold;">first paper</span> (other centers' press releases notwithstanding) describing the sequencing of an entire animal genome with next-generation sequencing technology.  We revisited our old friend, <a href="http://genome.wustl.edu/genome.cgi?GENOME=Caenorhabditis%20elegans"><span style="font-style:italic;">C. elegans</span></a>, and published the paper, <a href="http://www.nature.com/nmeth/journal/v5/n2/abs/nmeth.1179.html">Whole-genome sequencing and variant discovery in <span style="font-style:italic;">C. elegans</span></a>, in <a href="http://www.nature.com/nmeth/index.html">Nature Methods</a>.  Two different <span style="font-style:italic;">C. elegans</span> strains were sequenced using the Solexa Sequence Analyzer (now Illumina Genome Analyzer) in about a week.  Compare that to the years it took to sequencing the initial <span style="font-style:italic;">C. elegans</span> genome in the 1990's.  In doing several strains, we were able to test methodologies for detecting sequence variations (mutations).  In other words, just as the initial <span style="font-style:italic;">C. elegans</span> genome was used as a proof-of-concept project for <a href="http://www.genome.gov/10001772">The Human Genome Project</a>, this <span style="font-style:italic;">C. elegans</span> project was used as a proof-of-concept for medical sequencing projects to discover the variations that lead to disease using next-generation sequencing technologies.  All of the sequencing was done at The Genome Center while the analysis was done jointly between The Genome Center and <a href="http://bioinformatics.bc.edu/marthlab/Main_Page">Gabor Marth's lab</a> (a Genome Center alum).</p>
