---
layout: post
title: 1000 Genome SNPs released
date: 2008-12-24 08:49:14.000000000 -06:00
categories:
- genomics
tags:
- 1000 Genomes
- genomics
- informatics
- IT
- science
- wustl
status: publish
type: post
published: true
meta:
  _edit_last: '2'
author:
  login: dd
  email: dooling@gmail.com
  display_name: dd
  first_name: David
  last_name: Dooling
excerpt: !ruby/object:Hpricot::Doc
  options: {}
---
<p>The <a href="http://www.1000genomes.org/">1000 Genomes Project</a> has announced its initial release of <a href="http://en.wikipedia.org/wiki/Single_nucleotide_polymorphism">SNP</a> data from four of the individuals sequenced to high depth-of-coverage as part of the <a href="/2008/03/1000-genomes.html">second pilot project (trios)</a>.  Here is the announcement from Paul Flicek of <a href="http://www.ebi.ac.uk/">EBI</a> and the 1000 Genomes Data Coordination Center (and formerly of Washington University).</p>
<blockquote>
<p>Dear All,</p>
<p>I'm pleased to provide everyone a stocking stuffer in the form of the first release of data from the 1000 Genomes project.</p>
<p>The preliminary list of SNPs for 4 of the high coverage individuals are now available on the EBI and NCBI 1000 Genomes FTP sites. Instructions on how to access the data can be found at <a href="http://www.1000genomes.org">http://www.1000genomes.org</a>.</p>
<p>In addition, we have created a project specific genome browser to allow the data to be visualised in the context of genome annotations and data from other projects including the Venter and Watson genomes.  The browser is based on the Ensembl platform and is available at <a href="http://browser.1000genomes.org">http://browser.1000genomes.org</a>.  We will be making updates to the browser throughout January to ensure the 1000 Genomes data is visible by default and is easy to find (SNP tracks can now be found on the "Features" menu).  I welcome any comments, questions or suggestions that that you have about the workings of the browser.</p>
<p>A long list of people worked very hard to get this done and any attempt to mention people will certainly miss some.  However, I would like to specifically acknowledge Tom Blackwell, Goncalo Abecasis, Fiona Hyland, Zam Iqbal, Laura Clarke, Eugene Kulesha, Yuan Chen, Stephen Keenan, Fiona Cunningham, Justin Paschall, Martin Shumway,<br />
Hoda Kouri and Steve Sherry.</p>
<p>All the very best for the holiday season.</p>
<p>Paul Flicek
</p></blockquote>
<p>Obviously, the three 1000 Genomes pilot projects have been a massive undertaking that has strained not only the production centers, but the IT and informatics infrastructures of the production and analysis centers. To date, over 3.8 terabases (3.8&times;10<sup>12</sup> or 3.8 trillion bases which is equivalent to about 1270 <a href="http://en.wikipedia.org/wiki/Ploidy">haploid</a> human genomes) have been submitted as part of these pilot projects. The average <a href="/2008/06/whats-in-an-srf.html">SRF</a> file submitted to the <a href="/2008/05/short-read-archive.html">NCBI SRA</a> stored 50 bytes of information per base; so the amount of data submitted so far is nearly 200 TB! At current broadband rates in the United States, it would take nearly 10 years to download all of this data (those still using 1600 baud modems may want to request they ship you the data on hard drives). Did I mention these are just the <em>pilot</em> projects?</p>
