---
layout: post
title: Another rich white guy sequences own genome
tags:
- genomics
- Helicos
- science
published: true
---

<p><a href="http://www.hulu.com/watch/4191/saturday-night-live-the-continental"><img src="/assets/walken-continental.jpg" alt="The Continental" title="The Continental" width="225" height="169" class="alignright size-full wp-image-1464" /></a></p>
<p>Following in the <a href="http://www.technologyreview.com/biomedicine/19328/">dubious footsteps of J. Craig Venter</a>, Stephen Quake has sequenced his own genome using a technology he helped develop which was commercialized by a company he helped to found (lots of conflicts of interest there). The letter, <a href="http://www.nature.com/nbt/journal/vaop/ncurrent/abs/nbt.1561.html">Single-molecule sequencing of an individual human genome</a>, was published in Nature Biotechnology and details the sequencing of Quake's genome on the <a href="http://www.helicosbio.com/">Helicos</a> Genetic Analysis System (formerly Heliscope). There is a more accessible description of the letter in a <a href="http://www.bio-itworld.com/news/08/10/09/stephen-quake-personal-genome-single-molecule-sequencing.html">friendly article over at Bio-IT World</a> (although it does have some great quotes from the most honest guy in all of next-gen sequencing, Clive Brown).</p>
<p>The bottom line is that three researchers sequenced a human genome in four Helicos runs. They report that each run took one week but do not report the total amount of time the sequencing took (i.e., did they complete the project in a month or did sample preparation, instrument performance, etc. cause it to take much longer?). They report reagent costs that are on par with current second generation technology <em>fully loaded</em> costs (which are closer to $50,000, not the $250,000-$500,000 they report in their letter). Yes, you read that right, their reagent costs alone are on par with reagents, instrument depreciation, personnel, and IT infrastructure for second generation platforms. Quake does report in <a href="http://www.bio-itworld.com/news/08/10/09/stephen-quake-interview.html">an interview</a> that the amortized equipment cost was $10,000-$20,000. Assuming a straight line depreciation for the Helicos instrument, a $10,000-$20,000 depreciation over four one-week runs yields a $125,000-$250,000 annual depreciation. Their <a href="http://boston.bizjournals.com/boston/stories/2009/06/01/story14.html\"">first instrument was sold for $963,000</a> and they subsequently lowered the <a href="http://www.genomeweb.com/sequencing/helicos-slashes-instrument-price-25-percent-obtains-order-university-maryland">price by 25%</a>. So assuming their price now is $725,000 and the residual value of the instrument is $0 (not a bad assumption given the pace of change in this industry), they are estimating a life of their instrument of about three to six years. Again due to the pace of change of this industry, even three years may be generous.</p>
<p>As for sequence analysis, their ability to call <a href="http://www.ncbi.nlm.nih.gov/About/primer/snps.html">SNP</a>s is comparable to second generation platforms. Their ability to call <a href="http://en.wikipedia.org/wiki/Copy_number_variation">copy number variations (CNVs)</a> is perhaps better than second-generation sequencing platforms; likely due to the single-molecule nature of the Helicos system not requiring an amplification step that introduces bias. They did not even address indel detection. This is likely due to the fact that most of the Helicos sequencing errors are due to the incorporation of dark bases, leading to many false positive deletion events. It is also hard to detect indels with such short reads (average length of aligned reads was 32 bases). However the most notable thing about their analysis was how shallow it was. I suppose they thought it was more important to have only three people on the paper (they make a <strong>big</strong> deal about this, referring to it several times in the letter as the "democratization of sequencing") than to do thorough analysis. What you can take away from this is that you may be able to sequence a human genome with three people, but it takes a village of bioinformaticians to do the sequence data justice.</p>
<p> Daniel MacArthur over at Genetic Future has a <a href="http://scienceblogs.com/geneticfuture/2009/08/first_human_genome_sequenced_u.php">nice assessment of the publication</a> and summarizes nicely.<br />
<blockquote>This paper sets the bar pretty low for other third-generation sequencing contenders: it appears that formal entry into the human genome sequencing race merely requires generating a genome sequence of the standard that second-generation sequencers were achieving in early 2008, at the same price that they're charging right now. That's a fairly uninspiring goal.</p></blockquote>
<p>Nonetheless, <a href="http://www.google.com/finance?client=ob&q=NASDAQ:HLCS">Helicos stock (NASDAQ:HLCS)</a> has risen from <a href="http://www.tradingmarkets.com/.site/news/Stock%20News/2473379/">$0.60 close at the end of last week to about $1.20 now</a>, a 100% jump. I suppose it is not surprising that we have yet to learn from <a href="http://www.nytimes.com/2009/01/04/magazine/04risk-t.html">our past mistakes</a>.</p>
